### DISCUSSION TITLE
DataPoint Class

### DISCUSSION THREAD

**Discussion by ZichengLiang** [ZichengLiang/CSU11013-GroupB-Team9#4] [Ideas]
Daniel, Muireann, Oliver and I are resiponsible for DataPoint Class. We can post our findings and questions here.
Others are also welcomed to comment here for collaboration purposes!

**Comment by dannyelx21** [comment]
hi

**Comment by ZichengLiang** [comment]
Hi everyone! Please leave a comment here when you find somewhere that you can improve! (or create an issue

**Comment by Pikacrazy** [comment]
Hi just got the email where is problem 11 stated?

**Comment by ZichengLiang** [comment]
in the “issue” part of the repository

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by ZichengLiang** [comment]
@kapoor-ar 

Hi Aryan, just a small reminder that I've put your date-time formatter inside the CONSTANTS class, it doesn't change the functionality, just make codes look neater.

Currently the change is still in my branch, tell me if there'll be any problem.
